test/parallel/test-dotenv-node-options.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('node:assert');
+const { describe, it } = require('node:test');
+
+if (process.config.variables.node_without_node_options) {
+  common.skip('missing NODE_OPTIONS support');
+}
+
+const relativePath = '../fixtures/dotenv/node-options.env';
+
+describe('.env supports NODE_OPTIONS', () => {
+
+  it('should have access to permission scope', async () => {
+    const code = `
+      require('assert')(process.permission.has('fs.read'));
+    `.trim();
+    const child = await common.spawnPromisified(process.execPath,
+                                                [ `--env-file=${relativePath}`, '--eval', code ], {
+                                                  cwd: __dirname
+                                                });
+    // Test NODE_NO_WARNINGS environment variable
+    // `stderr` should not contain "ExperimentalWarning: Permission is an experimental feature" message
```

Comment:
This is the explanation for checking `stdout` and `stderr` to empty string. Since, the node options that's tested here has a `NODE_NO_WARNINGS` env variable, both stderr and stdout is an empty string.